### DISCUSSION TITLE
How do I specify the root dir for grep and find file? (in monorepo)

### DISCUSSION THREAD

**Discussion by yunyu950908** [LazyVim/LazyVim#3036] [Q&A]
Hi, I'm new to vim and lazyvim, and currently in trouble how to change the root dir in a monorepo, I need to grep keywords or find files in a specific project in a monorepo, but if I use `<space + />` to grep keywords, there're bunch of results from different projects. How do I specify the root dir to another dir? I've tried `:cd /path/to/dir ` but not work, it always use the folder where the .git folder placed. I need some help ...
<img width="716" alt="image" src="https://github.com/LazyVim/LazyVim/assets/25625252/1b37fd9d-79fa-407f-8a56-831f1351211f">

**Comment by dpetka2001** [comment]
The `root_dir` in LazyVim is defined by LSP servers themselves. You could use `<leader>sG`, which will only grep in files in your `cwd` instead.

**Reply by razvan** [reply]
I'm also interested in this. Searching in `cwd` doesn't work because lazy never changes the `cwd`.

**Reply by razvan** [reply]
> I'm also interested in this. Searching in cwd doesn't work because lazy never changes the cwd.

Never mind. Just after posting this I found out that it does indeed work if I explicitly change the wd before searching.

**Reply by guillenotfound** [reply]
How do you change the cwd? I know I can use `LazyRoot` to get information about it but IDK how to change it.
